### DISCUSSION TITLE
case-studies/periodsomnia/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#259] [Website Discussions]
# case-studies/periodsomnia/

For those that have never come across his work, Thomas Grove Carter is a bit of a legend in the Final Cut Pro Community.

https://fcp.cafe/case-studies/periodsomnia/

**Comment by florian-duffe** [comment]
Thank you, Thomas, so cool! Could you elaborate a bit on your usage of Video-Roles in this project? Am I guessing right that it's per camera? Did you have an extra keyword for VFX?
Also: FCP FTW is legendary!

**Reply by thomasgcarter** [reply]
All the Alexa was the Video Role. There was an additional Role for the Thermal and another for the period animation (called Medusa). 

The was initially for visual separation in the timeline, but also allowed me to quickly export EDLs for post which only contained the particular elements. Like omitting temp animation for the grade for example.
